package.json

```diff
@@ -20,7 +20,11 @@
   "main": "./dist/index.js",
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "dependencies": {},
+  "dependencies": {
+    "@types/storybook__addon-info": "^5.2.1",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^1.1.1"
```

Comment:
remove semantic-ui and semantic-ui-react

Reply:
done